system/modules/core/library/Contao/Model.php

```diff
@@ -258,6 +264,25 @@ public static function getPk()
 
 
 	/**
+	 * Return an array of unique field/column names
+	 * Do not include the PK here as this is handled separately
+	 *
+	 * @return array
+	 */
+	public static function getUniqueFields()
+	{
+		$objDca = \DcaExtractor::getInstance(static::getTable());
+
+		if ($objDca->hasUniqueFields())
```

Comment:
This can be omitted as `\DcaExtractor` is initializing to empty array.
Simply always return the value of `$objDca->getUniqueFields()`